src/editor/editorinteractive.h

```diff
@@ -116,6 +151,10 @@ class EditorInteractive : public InteractiveBase {
 		return nullptr;
 	}
 
+	bool omnipotent() const override {
```

Comment:
Unless you need this for some special distinction somewhere, override the `can_see`and `can_act` functions and then get rid of this.

Reply:
can_see and can_act are usually true for the interactive player, but we don't want normal players to be allowed to change the real fill of inputqueues and other editor-only in-detail configuration settings ;)

Reply:
Maybe rename it to "is_editor()" then so it will become immediately apparent what it is for?

Reply:
Perhaps debuggers will like to hack this function to always return true. Might be useful to investigate all kinds of nasty logic bugs. For that reason I'd prefer to keep the name "omnipotent" so it's clear what a change in this function will effect.